### DISCUSSION TITLE
Hide/pause work related notifications

### DISCUSSION THREAD

**Discussion by bric3** [community/community#29665] [Mobile]
So I'm using GitHub for my job and for personal open source projects, during off hours or holidays I'd like to hide PR or discussions that happen during that time, and be able to show whatever happened during that time when back to work.

Eg like a toggle for one or more organisation.

For example Slack has an option to **pause** notifications for a period.

![slack pause notification options](https://user-images.githubusercontent.com/803621/185352308-57d6cd8a-4e86-43a8-91f4-69011021aeaf.png)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.75.0 (74400986)
Device: iPhone 13 (iOS 16.0)
</details>

**Comment by mazkdevf** [comment]
you could basically unwatch all the repositories / what ever where you don't want them idk prbl not the best solution,

**Reply by bric3** [reply]
Yeah I know but it's not global enough there could be tens or more of watched repositories in a org. Also you'd miss what happened during that time. Be it holiday not simply different working hours in an international organization. / Company
